src/Helper.c

```diff
@@ -3,18 +3,67 @@
 #include <time.h>
 #include <math.h>
 
+// TODO: do we have to declare these two strucuts in this file and the header file too?
 struct Node {
   int data;
   struct Node* next;
 };
 
+typedef struct Element {
+  int number;
+  char letters[3];
+} Element;
+
+Element * initializeArrayOfElements(int size) {
+  time_t t;
+  srand((unsigned) time(&t));
+
+  // TODO maybe there is a better way to create a random string sequence?
```

Comment:
could also save a number between 65 and 90 (ASCII) as datatype char, random_char = 65 + (rand() % 26),
but I think this way is also totally fine.